libs/scraper-archive/src/getTestData.ts

```
@@ -0,0 +1,116 @@
+  if (!testFolder) {
+    return [];
+  }
+  const overrideData = getOverrideData();
+  const results: T[] = [];
+  const pattern = `${testFolder}/**/${recordTime}/**/${section}/**/*${searchPattern}*`
+  const matched = globSync(pattern);
+  for (const match of matched) {
+
+    const matchedFolder = path.dirname(match);
+    const machedFilename = path.basename(match);
+    const step = parseInt(machedFilename.split('-')[0]);
```

ESLint

[error] Missing radix parameter.

(radix)